src/store/peopleContext.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { createContext, useState } from 'react';
+import { Person } from '../types';
+import { getPeople } from '../utils/api';
+
+export const PeopleContext = createContext({
+  isError: false,
+  getPeopleFromServer: () => {},
```

Comment:
Consider defining `getPeopleFromServer` as a no-op function like `() => {}` in the default context value to prevent potential errors if it's called before the `PeopleProvider` is mounted.